### PR TITLE
[-] BO : fixed mysql 5.7 "virtual" is a reserved keyword

### PR DESCRIPTION
**Pull request by raphael-homann**
Please take the time to edit the "Answers" rows with the necessary information:

| Questions | Answers |
| --- | --- |
| Branch? | 1.6.1.x |
| Description? | AdminAttachmentsController fails with mysql 5.7 |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? |  |
| How to test? | go to AdminAttachmentsController with mysql 5.7 |

### CODE REVIEW

File: controllers/admin/AdminAttachmentsController.php [hunk 1]
```diff
@@ -53,7 +53,7 @@ public function __construct()
                 'align' => 'center',
                 'class' => 'fixed-width-xs'
             ),
-            'name' => array(
+            'name' => array(v
```
Comment (Quetzacoalt91) [inline controllers/admin/AdminAttachmentsController.php]: Hmm ... I guess we have a parse error here !

**Comment by raphael-homann** [comment]
oops, right, fixed it.

**Comment by kpodemski** [comment]
@xBorderie it's already cherry picked

**Comment by xBorderie** [comment]
Thanks @kpodemski !

**Pull request merged**